pymc3/tests/test_distributions_random.py

```diff
@@ -1257,7 +1256,6 @@ def ref_rand(size, mu, lam, alpha):
             ref_rand=ref_rand,
         )
 
-    @pytest.mark.xfail(reason="This distribution has not been refactored for v4")
```

Comment:
These tests that are using a `ref_rand` should be removed as well, unless there is a good reason to test them against a reference. If we are just copying the same scipy/numpy expression we use in our `random Ops` the default tests in `BaseTestDistribution` should be used instead.